packages/dataviz/package.json

```diff
@@ -35,27 +35,41 @@
   },
   "dependencies": {
     "classnames": "^2.2.6",
-    "d3": "^6.3.0",
+    "d3-color": "^2.0.0",
```

Comment:
no please this goes against our cdn infra.
Please keep to use d3.
By the way there is a PR for that: https://github.com/Talend/ui-scripts/pull/176/files
But target was more for external package than internal.

Reply:
Thanks for the help, wasn't aware of it. Did change it.